kin-devplatform-sdk/src/main/java/kin/devplatform/settings/presenter/SettingsPresenter.java

```diff
@@ -177,16 +184,48 @@ private void switchAccount(int accountIndex) {
 		accountManager.switchAccount(accountIndex, new KinCallback<Boolean>() {
 			@Override
 			public void onResponse(Boolean response) {
+				view.stopWaiting();
```

Comment:
check for null view

Reply:
I would expect some dialog/text that says that restore has finished successfully

Reply:
We can add one but Ayelet didn't say anything about it.
Will talk to her after finish with the PR and before merge.